src/main/java/org/apache/maven/plugins/artifact/buildinfo/ReferenceBuildinfoUtil.java

```diff
@@ -213,8 +214,10 @@ private ReproducibleEnv extractEnv(File file, Artifact artifact) {
             } else {
                 log.warn("no MANIFEST.MF found in jar " + file);
             }
+        } catch (ZipException e) {
+            log.warn("Corrupt jar file " + file + "\n" + e.getMessage());
         } catch (IOException e) {
-            log.warn("unable to open jar file " + file, e);
+            log.warn("unable to read jar file " + file + "\n" + e.getMessage());
```

Comment:
thinking again at it: if we do that, we need to display the exception class
like if the exception message was the first line of the stacktrace, as it brings useful info = the type of issue

with that pattern, ok, the rest of the stacktrace can be ignored